modules/plugins/ui/edgy-nvim/config.nix

```diff
@@ -0,0 +1,30 @@
+{
+  config,
+  lib,
+  ...
+}: let
+  inherit (lib.modules) mkIf;
+  inherit (lib.strings) optionalString;
+  inherit (lib.nvim.dag) entryBefore;
+  inherit (lib.nvim.lua) toLuaObject;
+
+  cfg = config.vim.ui.edgy-nvim;
+in {
+  config = mkIf cfg.enable {
+    vim = {
+      startPlugins = ["edgy-nvim"];
+      pluginRC.edgy-nvim = entryBefore ["basic"] ''
```

Comment:
This is incorrect, pluginRC doesn't have a section called `basic`. You can either:
1. Use `entryAnywhere`, which should be fine
2. Put the variables in `luaConfigRC` instead, and change `entryBefore` to `entryAfter`, since later takes precedence, and we want to be overriding the values in basic, if there are any.